### PR TITLE
Create Soldier and Bananas.cpp

### PR DESCRIPTION
**Pull request by anishnitjsr**

### CODE REVIEW

**Review by SR-Sunny-Raj — CHANGES_REQUESTED** [review]
Please also add the problem link along with your solution.

File: Coders/Competitive Programming/Codeforces/Soldier and Bananas.cpp [hunk 1]
```diff
@@ -0,0 +1,23 @@
+#soldier and bananas 
+
```
Comment (anishnitjsr) [inline Coders/Competitive Programming/Codeforces/Soldier and Bananas.cpp:2]: #https://codeforces.com/contest/546/problem/A

**Review by SR-Sunny-Raj — APPROVED** [review]
Please don't forget to star the repository. It would be highly appreciable.

**Comment by anishnitjsr** [comment]
how many pr i can make in your  github account ?

**Comment by SR-Sunny-Raj** [comment]
You can just make one valid PR in a single repository.

**Pull request merged**